composer.json

```diff
@@ -15,24 +15,26 @@
         }
     ],
     "require": {
-        "symfony/twig-bundle": "~2.1",
-        "symfony/framework-bundle": "~2.1",
-        "symfony/console": "~2.1",
+        "php": ">=5.3",
+        "symfony/twig-bundle": "~2.3|~3.0",
```

Comment:
@bmxjama  I took it from https://github.com/nelmio/NelmioApiDocBundle/blob/master/composer.json